SEP-0002-revised.md

```diff
@@ -0,0 +1,245 @@
+1. Project Members can call for an election either by informing the Steering Committee or the Ombudsperson.
+If a vote is called for then the candidate will be come a Project Member with a simple majority of all votes cast in the election.
+An election needs to be announced to the Project Members one week before the voting opens and voting will be open for two weeks.
+1. The candidate will be informed promptly of the outcome by the Steering Committee.
+
+The name of a candidate should not be shared outside of the Project Members until they become a Project Member.
+A public list of Project Members will be kept on the SunPy website.
+
+Nominations, approvals and any elections of Project Members should take place before the initial election of the Steering Committee.
+
+### Active and Emeritus Status
+
+Project Members have no term limits.
+Project Members who are no longer actively contributing to the project should contact the Steering Committee to move themselves to emeritus status.
+Active Project Members who have not made any significant contributions for two years, will be contacted by the Steering Committee and be asked to move themselves to emeritus status.
+The Steering Committee can change a Project Member's status to emeritus.
+Emeritus Project Members will continue to be listed on the website.
+Emeritus Project Members are not able to participate in votes.
+An emeritus member may request to be marked as active at any time if they feel they are making contributions.
+
+## Removing Project Members
+
+In exceptional circumstances, it may be necessary to remove someone from the Project Members against their will.
+A vote of the Project membership must be held to remove a Project Member.
+Such a vote is triggered by a motion (sent to the Steering Committee and the Ombudsperson) made by an active Project Member, which must be seconded by an additional Project Member.
+The vote must conclude no more than three months after the motion is seconded.
+Removal requires approval by two-thirds of all active Project Members at the time the motion is made.
+The motion, second, and vote will be by secret ballot.
+Removal under this provision will be reflected by updating the list of Project Members.
+A member removed via this mechanism does not have emeritus status.
+
+It may be necessary for the Ombudsperson and the Steering Committee to remove a Project Member for violations of the Code of Conduct.
+In this case, the Steering Committee, Ombudsperson and Board will work together to make this decision.
+
+# Ombudsperson
+
+The Ombudsperson represents the interests of the SunPy community by providing an alternative point of contact for sensitive issues such as code of conduct violations and ethical concerns.
+Candidates for this project role are publicly nominated by the Board, after which at least two weeks for comment is allowed, and then the nominee must be confirmed by two-thirds of the active Project Members.
```

Comment:
two-thirds is the same amount of votes needed to remove a project member which seems to define a high bar. Do we want it to be that hard to confirm an Ombudsperson?

Reply:
That's fair, and a high bar. Perhaps we should make this 2/3s of those voting to not enforce a turnout requirement?

Reply:
I have changed this.